Scripts/script-CommonServer.yml

```diff
@@ -982,9 +997,9 @@ script: |-
   }
 
   /**
-   * Get list data which is JSON format into an object that can be an array as well
```

Comment:
-   * Get list data which is in JSON format changed into an object that can be an array as well